zcbor/zcbor.py

```diff
@@ -981,11 +995,11 @@ def __init__(self, *args, **kwargs):
     # Name of variables and enum members for this element.
     def var_name(self, with_prefix=False):
         if with_prefix or not self.short_names:
-            name = ("_%s" % self.id(with_prefix=with_prefix))
+            name = ("%s" % self.id(with_prefix=with_prefix))
```

Comment:
There was f-string used for string substitution in different parts of the code. Is there any particular reason that `%` is used here?

Reply:
This substitution was unnecessary now, removed, thanks.